### ISSUE TITLE
Activity projektmunka

### ISSUE THREAD

**Issue by vicziani** [Dr-Tasko-Janos/training-solutions#1]
Megnéztem az Activity projektmunkát, rendben van. Ha jól látom, az óra megoldás alapján dolgoztál.

**Comment by Dr-Tasko-Janos** [comment]
Igen, mivel ehhez a projekthez érve már kezdett összecsúszni minden. Tekintettel arra, hogy már közeledik a zárás, a generális ismétlés keretében, a bonyolultabb feladatokat ismét meg fogom oldani, s köztük fog szerepelni eme projektmunka is. Úgy érzem, eljött az ideje egy átfogó ismétlések, hogy minden a helyére kerüljön stabilan, lévén azt vettem észre, hogy néhány korábban általam stabilan használt metódus megkopott bennem (például System.arraycopy(), egyes String metódusok) és egyre inkább elkényelmesedve kezdek hagyatkozni az Idea automatikus kódkiegészítésére ahelyett, hogy fejből jutna eszembe minden.
Nem szeretném ha meglepetésként érne a vizsgán, ha olyan fejlesztőeszközben kellene dolgoznom, amiben szándékosan inkativak bizonyos segéd funkciók s ne tudnék hirtelen beimportálni egy osztályt vagy kitalálni a megfelelő metódus nevét. Remélem nem teszem rosszul? :)

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by vicziani** [comment]
Ilyen nem lesz, IDEA-t lehet majd használni, szerintem csináld a gyakorlati feladatokat, és akkor maguktól rögzülnek ezek. Metódusneveket nem kell megtanulni, én sem tudom őket fejből. Csak azokat, amiket sokat használok.

**Comment by Dr-Tasko-Janos** [comment]
Rendben, köszönöm a tanácsot. 👌😀

Sent from ProtonMail mobile

-------- Original Message --------
